### ISSUE TITLE
Favicons too small sometimes.

### ISSUE THREAD

**Issue by TheSixtyFour** [tborychowski/newtab-bookmarks#19]
When developers use small favicons it sort of makes the whole thing look bad, maybe you could add a third option for icons using 'https://icon-fetcher-go.herokuapp.com/icon?size=64&url='

**Comment by tborychowski** [comment]
Hi,
Can you provide an example URL?
There are sites where authors don't bother with a proper icon and in such cases it's impossible to get one.

To your question - I'm reluctant to made my addon dependant on someone else's service.
I was actually using it initially, but it went down a couple of times and I had to devise my own.

If you can provide me with a couple of URLs that don't work with addon (showing a small ugly icon), and work with the icon-fetcher I would have a look.
Another option might be to host the icon-fetcher myself...

**Comment by TheSixtyFour** [comment]
Google, Google Docs, Wix sites and Wattpad.
Thanks for taking a look!

**Comment by tborychowski** [comment]
Hi,

So I've done some checking and there are a couple of things:
1. Google sucks immensely when it comes to favicons. (as with most of the stuff, they don't care about the standards if it doesn't suit them and they create their own) So it's very tricky to get their favicons.
2. I checked google stuff with icon-fetcher and for google docs I'm not getting the icon, but a letter:
![g-64-ea4335](https://user-images.githubusercontent.com/392513/51737899-b6230b80-2085-11e9-8861-94d889e27b68.png)
(and same for wix.com)
3. In the add-on - google stuff produces some icons, but not the best resolution
4. Some icons don't show up if you have firefox blocking enabled for the extension page:
<img width="371" alt="screenshot 2019-01-25 at 09 38 48" src="https://user-images.githubusercontent.com/392513/51738479-3007c480-2087-11e9-8b67-4574a9710bf0.png">
5. Even if i'd host the icon-fetcher service and use it, the issue is: when the addon requests a favicon and receives the response - it cannot tell what resolution it is (if it's an .ico file). So it cannot say, e.g.: Oh, I've got this icon, but it's low-res, I will fetch another...

So, to sum up, as a workaround I have a "whitelist" of urls that I'm using for the sites that don't easily provide the icon. I added a higher resolution google icons to there, and added wix and wattpad icons to the list, so you should get them. 
(please mind that the wix icon comes from some blog, not the official wix.com page, as they do not provide a hi-res icon anywhere! just the 16x16px favicon).

Let me know if that's sufficient for you.
And send on any other URLs that you'd like to see better icons :-)
